components/learn/modules/ROOT/pages/write-contracts.adoc

```diff
@@ -0,0 +1,158 @@
+= Write Solidity Contracts
+
+Welcome to the exciting world of smart contract development! This guide will let you get started writing Solidity contracts by going over the following:
+
+ * <<setting-up-a-solidity-project, Setting up a Solidity Project>>
+ * <<compiling-solidity-source-code, Compiling Solidity Source Code>>
+ * <<multi-file-projects, Multi-File Projects>>
+ * <<using-openzeppelin-contracts, Using the OpenZeppelin Contracts library>>
+
+=== About Solidity
+
+We won't cover language concepts such as syntax or keywords in this guide. For that, you'll want to check out the following curated content, which feature great learning resources for both newcomers and experienced developers:
+
+ * For a general overview of how Ethereum and smart contracts work, the official website hosts a https://ethereum.org/learn/[Learn about Ethereum] section with lots of beginner-friendly content
```

Comment:
```suggestion
 * For a general overview of how Ethereum and smart contracts work, the official website hosts a https://ethereum.org/learn/[Learn about Ethereum] section with lots of beginner-friendly content, including an https://studio.ethereum.org/[online IDE] to write your own smart contracts projects.
```

Reply:
I didn't include the IDE on purpose: not only do I find it confusing, but it also uses the Contracts library precisely the way the teach our users not to, by copy-pasting the code.